src/v2/Apps/Order/Components/__tests__/PaymentPicker.jest.tsx

```diff
@@ -194,6 +194,13 @@ describe("PaymentPickerFragmentContainer", () => {
     _mockStripe().createToken.mockImplementation(() =>
       Promise.resolve({ error: "bad error" })
     )
+    env.clearErrors()
```

Comment:
jest blocked using his hooks outside of test files, the easy refactor was to shift this control to the tests